package.json

```diff
@@ -13,7 +13,16 @@
     "watch": "node src/start watch",
     "lint": "eslint src -c .eslintrc --ext js",
     "lint:fix": "eslint src -c .eslintrc --fix --ext js",
-    "lint:changed": "LIST=`git diff-index --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi"
+    "lint:changed": "LIST=`git diff-index --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi",
+    "prepare": "husky install"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint:fix && yarn build"
+    }
+  },
```

Comment:
Do we need this since we already have the `pre-commit` script in the husky directory?

Reply:
You are right, this is from the old version of Husky